test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -511,25 +513,27 @@ public void PreferenceSetting()
 
             initalSetting.ConnectorType = ConnectorType.BEZIER;
             initalSetting.ConsoleHeight = 100;
-            initalSetting.IsBackgroundPreviewActive = true;
+            initalSetting.SetIsBackgroundPreviewActive(backgroundPreviewName, true);
 
             initalSetting.Save(tempPath);
```

Comment:
but shouldn't we also have a test that tries to load an old settings file with the old format?